CHANGELOG.md

```diff
@@ -1,3 +1,103 @@
+### citus v11.2.0 (January 30, 2023) ###
+
+* Adds support for outer joins having a recurring rel in the outer side of the
+  join (e.g., \<reference table\> LEFT JOIN \<distributed table\>)
+
+* Adds support for creating `PRIMARY KEY`s and `UNIQUE`/`EXCLUSION`/`CHECK`/
+  `FOREIGN KEY` constraints via `ALTER TABLE` command without providing a
+  constraint name
+
+* Adds support for using identity columns on Citus managed tables
+
+* Adds `citus_task_wait()` UDF to wait on desired task status
+
+* Adds `source_lsn`, `target_lsn` and `status` fields into
+  `get_rebalance_progress()`
+
+* Adds `citus_get_transaction_clock()` UDF that returns a monotonically
+  increasing logical causal clock for the current transaction
+
+* Allows `citus_update_node()` to work with nodes from different clusters
+
+* Introduces `citus.propagate_session_settings_for_loopback_connection` GUC to
+  propagate local settings to new connections
+
+* Introduces `citus_copy_shard_placement()` UDF with node id
+
+* Introduces `citus_move_shard_placement()` UDF with node id
+
+* Propagates `BEGIN` properties to worker nodes
+
+* Deprecates `citus.replicate_reference_tables_on_activate` and makes it
+  always `off`
+
+* Drops GUC `citus.defer_drop_after_shard_move`
+
+* Drops GUC `citus.defer_drop_after_shard_split`
+
+* Drops `SHARD_STATE_TO_DELETE` state and uses the cleanup records instead
+
+* Adds signal handlers for queue monitor to gracefully shutdown, cancel and to
+  see config changes
+
+* Defers cleanup after a failure in shard move or split
+
+* Extends cleanup process for replication artifacts
+
+* Improves a query that terminates compelling backends from
+  `citus_update_node()`
+
+* Includes Citus global pid in all internal `application_name`s
+
+* Reuses connections for shard splits and logical replication
```

Comment:
Isn't this already on 11.1? https://github.com/citusdata/citus/pull/6314

Reply:
correct